src/OpenLoco/Vehicles/CreateVehicle.cpp

```diff
@@ -655,9 +654,9 @@ namespace OpenLoco::Vehicles
 
     // 0x004B1E77
     // Free orderId?
-    static void sub_4B1E77(const uint16_t orderId)
+    static void sub_4B1E77(const RoutingHandle routing)
     {
-        uint16_t baseOrderId = (orderId & ~(0x3F)) / max_num_routing_steps;
+        const uint16_t baseOrderId = routing.getVehicleRef();
```

Comment:
should go through and relabel anything routing related that is currently orderId and make it routingId related.